platform/android/export/export.cpp

```diff
@@ -522,8 +522,8 @@ void EditorExportPlatformAndroid::_fix_resources(Vector<uint8_t>& p_manifest) {
 	const int UTF8_FLAG = 0x00000100;
 	print_line("*******************GORRRGLE***********************");
 
-	uint32_t header = decode_uint32(&p_manifest[0]);
-	uint32_t filesize = decode_uint32(&p_manifest[4]);
+	// uint32_t header = decode_uint32(&p_manifest[0]);
```

Comment:
I'm honestly not sure about removing those, that's why I just commented them out, which I know @akien-mga hates :) They could be helpful in the future, even if currently unused. I'm waiting for input